src/main/java/com/commercetools/service/paypalPlus/impl/PaypalPlusPaymentServiceImpl.java

```diff
@@ -54,6 +61,47 @@ public PaypalPlusPaymentServiceImpl(@Nonnull APIContextFactory paypalPlusApiCont
         return paymentStageWrapper((paypalPlusApiContext) -> Payment.get(paypalPlusApiContext, paymentId));
```

Comment:
pls, unwrap all redundant braces () in the lambdas.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/57/commits/b81b95c0791f0943faa367148ce26a5bbe339d85